### PR TITLE
DDT-753: move methods and models from SI

### PR DESCRIPTION
**Pull request by mp-assco**

### CODE REVIEW

File: src/main/java/org/dcsa/core/events/service/impl/ShipmentEquipmentServiceImpl.java [hunk 1]
```diff
@@ -117,6 +159,41 @@
...
+    // TODO: we have a known bug here that needs to be addressed.
+    //  carrierBookingReference can differ from each cargoItem.
+    return shipmentRepository
+        .findByCarrierBookingReference(carrierBookingReference)
+        .switchIfEmpty(
+            Mono.error(
+                ConcreteRequestErrorMessageException.invalidParameter(
+                    "No shipment found with carrierBookingReference: " + carrierBookingReference)))
+        .flatMap(
+            x ->
```
Comment (pimg) [inline src/main/java/org/dcsa/core/events/service/impl/ShipmentEquipmentServiceImpl.java:176]: use more appropriate name for example shipment

File: src/main/java/org/dcsa/core/events/service/ShipmentEquipmentService.java [hunk 2]
```diff
@@ -10,4 +11,8 @@
 	Mono<List<ShipmentEquipmentTO>> createShipmentEquipment(UUID shipmentID, String shippingInstructionID, List<ShipmentEquipmentTO> shipmentEquipments);
 
 	Mono<List<ShipmentEquipmentTO>> findShipmentEquipmentByShipmentID(UUID shipmentID);
+
+	Mono<List<ShipmentEquipmentTO>> insertShipmentEquipmentTOs(List<ShipmentEquipmentTO> shipmentEquipments, ShippingInstructionTO shippingInstructionTO);
```
Comment (pimg) [inline src/main/java/org/dcsa/core/events/service/ShipmentEquipmentService.java:15]: method name seems inappropriate. It has TO in the name whereas the others do not. Also the name insert feels ambiguis to create. "addShipmentEquipmentToShippingInstruction seems better

**Review by pimg — CHANGES_REQUESTED** [review]

File: src/main/java/org/dcsa/core/events/service/impl/ShipmentEquipmentServiceImpl.java (hunk 1 above)
Comment (mp-assco) [reply src/main/java/org/dcsa/core/events/service/impl/ShipmentEquipmentServiceImpl.java]: Renamed to `shipment`.

File: src/main/java/org/dcsa/core/events/service/ShipmentEquipmentService.java (hunk 2 above)
Comment (mp-assco) [reply src/main/java/org/dcsa/core/events/service/ShipmentEquipmentService.java]: Renamed `insertShipmentEquipmentTOs` to `addShipmentEquipmentToShippingInstruction`.

**Review by pimg — APPROVED** [review]

**Pull request merged**